### PR TITLE
Upbound Universal Crossplane Addon for EKS Managed Addon

### PR DESCRIPTION
**Pull request by elamaran11**
*Issue #, if available:*

*Description of changes:*

With respect to the Issue #318 ro build an addon for CrossPlane, this PR addresses the creation for Upbound Universal Crossplane addon using EKS Managed addon.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/addons/upbound-universal-crossplane.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+## Configuration Options
+
+- `version`: Version of the Upbound Universal Crossplane add-on to be installed. 
+
+```bash
+# Command to show versions of the UXP add-on
+aws eks describe-addon-versions 
+    --addon-name upbound_universal-crossplane \
+    --kubernetes-version 1.23 \
```
Comment (shapirov103) [inline docs/addons/upbound-universal-crossplane.md:36]: we are on 1.24 now.

File: examples/blueprint-construct/index.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ export interface BlueprintConstructProps {
 export default class BlueprintConstruct {
     constructor(scope: Construct, props: cdk.StackProps) {
 
-        blueprints.HelmAddOn.validateHelmVersions = true;
+        blueprints.HelmAddOn.validateHelmVersions = false;
```
Comment (shapirov103) [inline examples/blueprint-construct/index.ts:26]: was it intentional?

**Review by shapirov103 — COMMENTED** [review]
Looks great, a couple of minor comments.

File: docs/addons/upbound-universal-crossplane.md (hunk 1 above)
Comment (elamaran11) [reply docs/addons/upbound-universal-crossplane.md:36]: ### Fixed.

File: examples/blueprint-construct/index.ts (hunk 2 above)
Comment (elamaran11) [reply examples/blueprint-construct/index.ts:26]: No it is not. Used it for my testing. Reverted back

**Comment by elamaran11** [comment]
@shapirov103 All comments are fixed. Please run `e2e`.

**Review by shapirov103 — DISMISSED** [review]
LGTM

**Comment by shapirov103** [comment]
@elamaran11 getting this error:

`blueprint-construct-dev \|  86/216 \| 6:01:33 AM \| CREATE_FAILED        \| AWS::EKS::Addon                       \| upbound_universal-crossplane-addOn (upbounduniversalcrossplaneaddOn) Resource handler returned message: "Addon specified is not supported (Service: Eks, Status Code: 400, Request ID: 9c48bd59-013d-45cc-9f91-22363634444e)" (RequestToken: 1050d10c-13b8-84f5-1257-b0ab55c385f5, HandlerErrorCode: InvalidRequest)`

**Comment by shapirov103** [comment]
Failed, presumably due to missing subscription to the Upbound add-on. Added subscription.

**Comment by shapirov103** [comment]
@elamaran11 please merge from main, there is a conflict now.

**Comment by elamaran11** [comment]
> @elamaran11 please merge from main, there is a conflict now.

@shapirov103 I have fixed the merge conflict now. Please proceed with e2e after the subscription is active for 1.24

**Review by shapirov103 — APPROVED** [review]

**Pull request merged**